docs/mine/connectivity.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Improving connectivity
+description: Tips and tricks for improving a miner's connectivity to the Filecoin network.
+---
+
+
+# Improving connectivity
+
+Filecoin miners, like participants in all peer-to-peer protocols, require a steady and quality pool of peers to communicate with in order to perform their various functions. For other participants on the network to establish incoming P2P connections with a miner, a few conditions must be met:
+
+* The miner's public IP address must be known
+* The protocol (TCP/UDP) and port number (0-65535) on which the miner is listening must be known 
+* All routers & firewalls must be configured to allow incoming traffic on that protocol/port combination
+
+The following steps are highly recommended for all miners who wish to successfully accept storage and retrieval deals.
+
+## Checking peer count
+
+To ensure storage and retrieval deals operate smoothly, it is recommended to check how many peers a miner is connected to after each start-up. In the Lotus client, a manual peer check can be performed with the command:
+
+```
+lotus-miner net peers
+``` 
+
+If a very low peer count is present (1-5), it is possible to manually connect the miner to the DHT by utilising one of the bootstrap peers listed in the branch's `./build/bootstrap/bootstrappers.pi` file.
```

Comment:
It'd be good to include the command here (`lotus net connect <address from bootstrappers>`).

Reply:
Thanks for the suggestion @arajasek! Since this PR already merged, @har00ga can you address in a follow-up?

Reply:
Can do!